### PR TITLE
Fix the issue Last Accessed Value is not displayed in session details table

### PR DESCRIPTION
**Pull request by sandushi**
### Purpose
fix https://github.com/wso2/product-is/issues/15076

### Changes from the PR
Move the table body content to the document ready function

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp [hunk 1]
```diff
@@ -204,6 +190,22 @@
...
+                var tableRow =
+                    "<tr>"
+                    + "<td><e:forHtmlContent value="${loop.index + 1}"/></td>"
+                    + "<td><e:forHtmlContent value="${session[2]}"/></td>"
+                    + "<td><e:forHtmlContent value="${session[3]}"/></td>"
+                    + "<td id='<e:forHtmlAttribute value="${session[1]}"/>'>" + accessedTime + "</td>"
+                    + "<td><input type='checkbox' onchange='toggleMasterCheckbox()' value='<e:forHtmlAttribute value="${session[0]}"/>' name='sessionsToTerminate' checked/></td>"
+                    + "</tr>";
+
+                $("#session-details tbody").append(tableRow)
```
Comment (Avarjana) [inline apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp:205]: Let’s add the semicolon here.

Comment (sandushi) [reply apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp]: Resolved by https://github.com/wso2/identity-apps/pull/3612/commits/17b1be24446142e8c7af2dc9faa3d4ad1392035f

File: apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp [hunk 2]
```diff
@@ -204,6 +190,22 @@
...
+                var tableRow =
+                    "<tr>"
+                    + "<td><e:forHtmlContent value="${loop.index + 1}"/></td>"
+                    + "<td><e:forHtmlContent value="${session[2]}"/></td>"
+                    + "<td><e:forHtmlContent value="${session[3]}"/></td>"
+                    + "<td id='<e:forHtmlAttribute value="${session[1]}"/>'>" + accessedTime + "</td>"
+                    + "<td><input type='checkbox' onchange='toggleMasterCheckbox()' value='<e:forHtmlAttribute value="${session[0]}"/>' name='sessionsToTerminate' checked/></td>"
+                    + "</tr>";
+
+                $("#session-details tbody").append(tableRow;
```
Comment (Avarjana) [inline apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp:205]:
```suggestion
                $("#session-details tbody").append(tableRow);
```

Comment (sandushi) [reply apps/authentication-portal/src/main/webapp/handle-multiple-sessions.jsp]: Resolved by https://github.com/wso2/identity-apps/pull/3612/commits/677371021d62d17835888feeba1a0b0cc053c324

**Review by thivi — APPROVED** [review]

**Review by thanujalk — APPROVED** [review]

**Pull request merged**